frontend/tools-page.html

```diff
@@ -94,159 +94,159 @@
         <div class="row">
             <div class="col l4 m6 s12">
                 <div class="card">
-                    <img class="card-img-top" src="images/google-search.jpg" alt="Card image cap">
+                    <img class="card-img-top" src="images/google-search.jpg" alt="Google Search">
                     <div class="card-block">
                         <h4 class="card-title">Google Search</h4>
                         <p class="card-text">Research with precision. Simple tools and tips to help you get better results, faster.</p>
-                        <a href="#" class="card-link">GO TO TOOL</a>
+                        <a href="https://www.google.com.ng/" class="card-link">GO TO TOOL</a>
```

Comment:
@andela-iakande remember to update this page as well.

Reply:
@chege I have fixed it.